nemo/lightning/io/fdl_torch.py

```diff
@@ -110,6 +112,8 @@ def enable():
     def _modified_serialize(self, value, current_path, all_paths=None):
         if isinstance(value, types.BuiltinFunctionType):
             return self._pyref(value, current_path)
+        if isinstance(value, partial) and not value.args:
```

Comment:
Wouldn't we be able to handle value.args as well?

Reply:
Thanks for the review! I added support to ` value.args`. Also, as discussed, I added a unit test that saves and re-load a TrainerContext with a `partial` function that has positional and keywords args.